vm/src/stdlib/random.rs

```diff
@@ -3,54 +3,33 @@
 use rand::distributions::Distribution;
 use rand_distr::Normal;
 
-use crate::function::PyFuncArgs;
-use crate::obj::objfloat;
 use crate::pyobject::{PyObjectRef, PyResult};
 use crate::vm::VirtualMachine;
 
 pub fn make_module(vm: &VirtualMachine) -> PyObjectRef {
     let ctx = &vm.ctx;
 
     py_module!(vm, "random", {
-        "guass" => ctx.new_rustfunc(random_gauss),
+        "guass" => ctx.new_rustfunc(random_normalvariate), // TODO: is this the same?
```

Comment:
I see a typo here, not introduced by you, but maybe a good time to fix it: `guass` -> `gauss`.